test/unit/test-purifier.js

```diff
@@ -492,6 +493,41 @@ function runSanitizerTests() {
       expect(purifyTagsForTripleMustache(html)).to.be.equal(html);
     });
 
+    it('should sanitize formatting related elements', () => {
+      const html = '<b>abc</b><img><div>def</div>'
+          + '<br><code></code><del></del><em></em>'
+          + '<i></i><ins></ins><mark></mark><s></s>'
+          + '<small></small><strong></strong><sub></sub>'
+          + '<sup></sup><time></time><u></u><hr>';
+      expect(purifyTagsForTripleMustache(html)).to.be.equal(
+          '<b>abc</b><div>def</div>'
+          + '<br><code></code><del></del><em></em>'
+          + '<i></i><ins></ins><mark></mark><s></s>'
+          + '<small></small><strong></strong><sub></sub>'
+          + '<sup></sup><time></time><u></u><hr>'
```

Comment:
Pretty hard to tell the difference between the expected and actual output here.

Reply:
cleaned up. Thanks!